automatic/kubernetes-cli/update.ps1

```diff
@@ -19,15 +19,16 @@ function global:au_SearchReplace {
     }
 
     "$($Latest.PackageName).nuspec" = @{
-      "(\<releaseNotes\>).*?(\</releaseNotes\>)" = "`${1}$($Latest.ReleaseNotes)`${2}"
+      "(\<copyright\>.*?)\d{4}(.*?\<\/copyright\>)" = "`${1}$($Latest.ReleaseYear)`${2}"
```

Comment:
Can you guarantee that this will guarantee that the copyright will be equal to the copyright by the Software Authors?

If not, then this should be a manual process.